src/app/models/resources/highlight-add.ts

```diff
@@ -19,4 +19,5 @@ export interface HighlightAdd {
   projectId: number;
   startDate?: Date;
   endDate?: Date;
+  description?: string;
```

Comment:
```suggestion
  description: string;
```
The description should not be nullable